packages/web/app/components/PatientPrinting/modals/SendToPatientModal.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useCallback } from 'react';
+import { toast } from 'react-toastify';
+
+import { useSendPatientPortalRegistrationEmail } from '../../../api/mutations/useSendPatientPortalRegistrationEmailMutation';
+import { EmailAddressConfirmationForm } from '../../../forms/EmailAddressConfirmationForm';
+import { TranslatedText } from '../../Translation';
+import { FormModal } from '../../FormModal';
+import { ModalGenericButtonRow } from '../../ModalActionRow';
+import { FormSubmitCancelRow } from '../../ButtonRow';
+
+export const SendToPatientModal = React.memo(({ open, onClose, patient }) => {
+  const { mutate: sendPatientPortalRegistrationEmail } = useSendPatientPortalRegistrationEmail({
+    onSuccess: () => {
+      toast.success(
+        <TranslatedText
+          stringId="patientDetails.resources.patientPortalRegistration.modal.sendToPatient.success"
+          fallback="Patient portal link successfully sent to patient"
+        />,
+      );
+      onClose();
+    },
+  });
+
+  const handleSubmit = useCallback(
```

Comment:
it's a bit subjective but I recommend to not memoize simple functions. Here is some background
https://beyondessential.slack.com/archives/C01RX8X3LT0/p1629765138003200